src/main/java/net/neoforged/moddevgradle/internal/ModDevPlugin.java

```diff
@@ -1,100 +1,33 @@
 package net.neoforged.moddevgradle.internal;
 
```

Comment:
Once we agree on everything else, this needs an extensive entry in the breaking changes document.